app/views/compute_resources_vms/show.html.erb

```diff
@@ -1,3 +1,18 @@
+<%= breadcrumbs(
+    items:
+    [
+      {
+        caption: _("Compute Resouce Vms"),
+        url: url_for(compute_resource_vms_path(@compute_resource))
+      },
+      {
+        caption: @vm.name
+      }
+    ],
```

Comment:
Following our discussion yesterday, the breadcrumbs on this page should be:
Compute Resources -> <compute resource name> -> VMs -> <vm name>
which is the same as in the CR images page.
The vms index page should also include similar breadcrumbs.